spyglass/src/main/java/com/linkedin/android/spyglass/ui/MentionsEditText.java

```diff
@@ -330,10 +498,10 @@ private boolean onCursorChanged(final int index) {
             return false;
         }
 
-        // Deselect any spans if the cursor is not at the end of it
         MentionSpan[] allSpans = text.getSpans(0, text.length(), MentionSpan.class);
         for (MentionSpan span : allSpans) {
-            if (span.isSelected() && text.getSpanEnd(span) != index) {
+            // Deselect span if the cursor is not on the span.
+            if (span.isSelected() && (index > text.getSpanEnd(span) || text.getSpanStart(span) > index)) {
```

Comment:
I think this would be slightly easier to read if you flipped the boolean around as: `(index < text.getSpanStart(span) || index > text.getSpanEnd(span))`